esp-wifi/src/preempt/timer/mod.rs

```diff
@@ -0,0 +1,11 @@
+use esp_hal::sync::Locked;
+
+#[cfg_attr(any(esp32, esp32s2, esp32s3), path = "xtensa.rs")]
```

Comment:
Hmm don't we have `#[cfg(xtensa)]` and `#[cfg(riscv)]` in esp-wifi?

Reply:
ah we use esp-metadata - so yes

Reply:
I pushed a fix.